### PR TITLE
Wording tweaks to VM PVC areas

### PR DESCRIPTION
**Pull request by andybraren**
Just some minor wording tweaks, fixes, and consistency adjustments to consider. WDYT @matthewcarleton @rawagner?

Note: I confirmed with PatternFly that there should be one `l` in `Canceling` to align with the corporate style guide - the [Terminology page](https://www.patternfly.org/v4/design-guidelines/content/terminology) will be updated soon.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form-status.tsx [hunk 1]
```diff
@@ -138,15 +138,15 @@ const UploadingStatus: React.FC<UploadingStatusProps> = ({
               className="kv--create-upload__alert"
               isInline
               variant={AlertVariant.warning}
-              title="Please don’t close this browser tab"
+              title="Please don't close this browser tab"
             >
-              Closing it will cause the upload to fail. You may still navigate the console.
+              Closing this tab will cause the upload to fail. Please wait until it completes.
```
Comment (matthewcarleton) [inline frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form-status.tsx:143]: The purpose of this text is to let the user know they can leave the flow and they won't lose their upload. We don't want to force them to stare at this page if they don't want to :) Maybe there's a better way to word that?

File: frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx [hunk 2]
```diff
@@ -376,8 +376,8 @@ export const UploadPVCForm: React.FC<UploadPVCFormProps> = ({
               inputID="request-size-input"
             />
             <p className="help-block" id="request-size-help">
-              Ensure your PVC size covers the requirements of the uncompressed image and any other
-              space requirements
+              Ensure that the Persistent Volume Claim is larger than the uncompressed image and meets any other
```
Comment (matthewcarleton) [inline frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx:379]: I prefer `covers the requirements` it's less open ended. Saying `is larger` might beg the question : How much larger? I am also a little torn on PVC vs Persistent Volume Claim - at this point the user likely understands what a PVC is and doesn't need it in full form.

File: frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-popover.tsx [hunk 3]
```diff
@@ -64,7 +64,7 @@ export const UploadPVCPopover: React.FC<PVCUploadStatusProps> = ({ pvc }) => {
       case UPLOAD_STATUS.UPLOADING:
         return {
           title: `Uploading`,
-          body: 'Please do not close this window, you can keep navigating the app freely.',
+          body: "Please don't close this browser tab until the upload is complete.",
```
Comment (matthewcarleton) [inline frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-popover.tsx:67]: same thing as above - I like educating the user on their ability to navigate around. "App" probably isn't the best word though. Maybe 
`Please don't close this browser tab until the upload is complete. You may still navigate the console freely.`

**Review by matthewcarleton — COMMENTED** [review]
Thanks @andybraren !
Most of these look like nice improvements to me. Added a few additional thoughts.

**Comment by andybraren** [comment]
Killing this ancient and likely outdated PR before the robots do. /close

**Pull request closed without merging**